### PR TITLE
Dropdown menu styling

### LINKED ISSUE #9305: Incorrect states on dropdown menus
Dropdown menus have incorrect styling when hovering menu items.

**Current Implementation**
image

**Mock-Up Fixes**
image

### PR DESCRIPTION
**Pull request by perkyguy**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #9305 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
I changed the `.active` style to have an `:after` svg icon (which I added as well), then changed the `:hover` to match the other dropdown items. 

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID: e8be8049-ac14-401f-89ff-ce0aaa3445b4

### CODE REVIEW

File: website/client/assets/scss/dropdown.scss [hunk 1]
```diff
@@ -40,10 +40,31 @@
...
     background-color: rgba(#d5c8ff, 0.32);
     color: $purple-200;
   }
+
+  &.active {
+    background-color: inherit;
+    color: $purple-200;
+    
+    &:active, &:hover, &:focus {
+      background-color: rgba(#d5c8ff, 0.32);
```
Comment (paglias) [inline website/client/assets/scss/dropdown.scss]: can you change this to use a color variable from scss/colors.scss? thanks

**Review by paglias — CHANGES_REQUESTED** [review]

File: website/client/assets/scss/dropdown.scss (hunk 1 above)
Comment (perkyguy) [reply website/client/assets/scss/dropdown.scss]: Is that `$suggested-item-color`? Or should it be something in the `$purple-XXX` family? I had used the equivalent of `$suggested-item-color` as that is what was used (hardcoded) before. Mainly asking if it should change based on the mockup provided in the issue

**Comment by perkyguy** [comment]
@paglias I made that change

**Comment by paglias** [comment]
LGTM but I haven't live tested (we should check other dropdowns like those in the TasksV2 PR aren't affected by these changes)

**Review by paglias — APPROVED** [review]

**Comment by Tressley** [comment]
This looks good with the exception of the active state being applied to the Navigation. Once we fix that, I think we're good to go.

image

**Comment by paglias** [comment]
hey @perkyguy how are you doing on this?

**Comment by perkyguy** [comment]
Sorry, had a bunch of things going on, hadn't gotten back to this. I'll take care of this soon. I know I was just having some issues getting my dev environment to work correctly (all of the menus were broken). I'll get to this (and my other PRs) fixed and resubmitted soon!

**Comment by paglias** [comment]
How is this going @perkyguy ? Let me know if you need any help here

**Comment by lemoness** [comment]
I've updated the labels to indicate that this PR has been abandoned and needs to be reviewed by our devs.

**Pull request closed without merging**